lib/screens/home_screen.dart

```diff
@@ -52,7 +66,10 @@ class _HomeScreenState extends State<HomeScreen> {
             ListTile(
                 onTap: () {
                   Navigator.pop(context);
-                  //TODO:open bottomsheet showing device images using storage_path package
+                  setState(() {
+                    model = _galleryImageFile[1];
```

Comment:
Let user choose  which directory they want. make a drop down in bottom sheet or somewhere in the ui